### PR TITLE
Milestone-5-interaction

### PR DESCRIPTION
**Pull request by nriqu322**
I add the interaction between components.

- Add a handleClick function to take numbers and operations as an input.
- Take the inputs and calculate the result to display in the calculator as a string.
- Take into consideration the error when dividing by zero.

### CODE REVIEW

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -1,38 +1,48 @@
...
-      <Button name="2" color="#e0e0e0" />
-      <Button name="3" color="#e0e0e0" />
-      <Button name="+" />
-    </div>
-    <div className="group">
-      <Button name="0" wide color="#e0e0e0" />
-      <Button name="." color="#e0e0e0" />
-      <Button name="=" />
+const ButtonPanel = props => {
+  // const handleClick = btnName => props.clickHandler(btnName);
```
Comment (suretrust) [inline src/components/ButtonPanel.js]: Let's remove commented out code.

File: src/logic/operate.js [hunk 2]
```diff
@@ -22,10 +22,14 @@ const operate = (numberOne, numberTwo, operation) => {
   }
```
Comment (suretrust) [inline src/logic/operate.js:22]: - [x] The `%` needs some more work, it should be treated as `percentage`, not `modulus`, e.g. 95 % === 0.95.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @nriqu322, 👋 

### Great job thus far! 🙌 Please see my comments. 👇 

**KIndly submit another CRR once these issues have been solved.** _Happy coding!_ 🎉

File: src/components/ButtonPanel.js (hunk 1 above)
Comment (nriqu322) [reply src/components/ButtonPanel.js]: done

File: src/logic/operate.js (hunk 2 above)
Comment (nriqu322) [reply src/logic/operate.js:22]: solved

**Review by shloch — APPROVED** [review]
## Hello @nriqu322 , 

Corrections made and project still works fine
calc

Thumbs up
Happy coding

**Pull request merged**